pom.xml

```diff
@@ -21,23 +21,68 @@
         <version>2.1.5.RELEASE</version>
     </parent>
 
-    <groupId>replication</groupId>
-    <artifactId>replication</artifactId>
+    <name>Replication :: Service</name>
+    <groupId>com.connexta.replication</groupId>
+    <artifactId>replication-service</artifactId>
```

Comment:
❓ Curious if `replication-service` was chosen for a reason?

Reply:
Was following the example of the transform team. Replication is providing a service so I thought it would be ok to follow the pattern.